app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,30 @@
+<nav class="navbar is-dark" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="/">
+      <i class="fas fa-dollar-sign is-size-5"></i>
+      <h1 class="has-text-weight-semibold has-text-white-ter">Budget Tracker</h1>
+    </a>
+
+    <a role="button" id="navbar-burger" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+      <%= link_to 'Purchases', purchases_path, class:"navbar-item" %>
+
+      <%= link_to 'Summaries', "/summary", class:"navbar-item" %>
+
+      <%= link_to 'Categories', "/category", class:"navbar-item" %>
+    </div>
```

Comment:
What about using the Rails path helpers for Summaries and Categories, as you've done above with Purchases?